src/rosdiscover/ros2_translator/arch.py

```diff
@@ -0,0 +1,41 @@
+import yaml
+from launch.actions import IncludeLaunchDescription
+from launch_ros.actions import Node
+from launch.launch_description_sources import get_launch_description_from_python_launch_file
+
+description = get_launch_description_from_python_launch_file('robot.launch.py')
```

Comment:
Let's also move these two lines `description` and `nodes` to the body of the if-statement at the bottom of the file.